extension/task/task.json

```diff
@@ -149,6 +226,16 @@
       "helpMarkDown": "The raw Personal Access Token for accessing GitHub repositories. Use this in place of `gitHubConnection` such as when it is not possible to create a service connection."
     },
 
+    {
+      "name": "securityUpdatesOnly",
+      "type": "boolean",
+      "groupName": "security_updates",
+      "label": "Perform security updates only.",
+      "defaultValue": "false",
+      "required": false,
+      "helpMarkDown": "If `true`, only security updates will be processed. Defaults to `false`.",
+      "visibleRule": "command!=update_script"
+    },
```

Comment:
There is an existing way to get this condition. Let us keep it so that we are very close to the official implementation.

Reply:
Good point, I have changed this to follow the official implementation